src/platforms/javascript/common/configuration/tree-shaking/index.mdx

```diff
@@ -36,7 +36,49 @@ To make optional code eligible for tree shaking, you can replace various flags i
 : Replacing this flag with `false` will tree shake all code in the SDK that is related to performance monitoring.
 **Attention:** `__SENTRY_TRACING__` must not be replaced with `false` when you're using any performance monitoring-related SDK features (e.g. `Sentry.startTransaction()`). This flag is intended to be used in combination with packages like `@sentry/next` or `@sentry/sveltekit`, which automatically include performance monitoring functionality.
 
-<PlatformSection notSupported={["javascript.nextjs"]}>
+`__RRWEB_EXCLUDE_CANVAS__`
+
+: Replacing this flag with `true` will tree shake all code in the SDK that is related capturing canvas elements with Session Replay. This is only relevant when using [Session Replay](../../session-replay/). You can enable this flag if you don't have any canvas elements on your page you care to record.
+
+`__RRWEB_EXCLUDE_IFRAME__`
+
+: Replacing this flag with `true` will tree shake all code in the SDK that is related capturing iframe content with Session Replay. This is only relevant when using [Session Replay](../../session-replay/). You can enable this flag if you don't have any iframes on your page you care to record.
+
+`__RRWEB_EXCLUDE_SHADOW_DOM__`
+
+: Replacing this flag with `true` will tree shake all code in the SDK that is related capturing shadow dom elements with Session Replay. This is only relevant when using [Session Replay](../../session-replay/). You can enable this flag if you don't have any shadow dom elements on your page you care to record.
+
+<PlatformSection notSupported={["javascript.nextjs", 'javascript.sveltekit', 'javascript.remix']}>
+
+### Tree Shaking Optional Code With Sentry Bundler Plugins
+
+If you are using one of our bundler plugins, you can use the `bundleSizeOptimizations` configuration option to tree shake optional code:
+
+```javascript
+sentryPlugin({
+  // other config
+  excludeDebugStatements: true,
+  excludePerformanceMonitoring: true,
+  excludeReplayCanvas: true,
+  excludeReplayIframe: true,
+  excludeReplayShadowDom: true,
```

Comment:
We are missing the surrounding `bundleSizeOptimizations` here.

Reply:
oops, damn! 😅